packages/itmat-ui-react/src/components/login/register.tsx

```diff
@@ -1,19 +1,30 @@
 import * as React from 'react';
-import { useMutation } from '@apollo/client/react/hooks';
+import { useQuery, useMutation } from '@apollo/client/react/hooks';
 import { NavLink, useHistory } from 'react-router-dom';
-import { CREATE_USER } from 'itmat-commons';
+import { GQLRequests, Models, CREATE_USER } from 'itmat-commons';
 import { Input, Form, Button, Alert, Checkbox, Select } from 'antd';
 import css from './login.module.css';
-import { sites } from '../datasetDetail/tabContent/files/fileTab';
+const {
+    GET_ORGANISATIONS
+} = GQLRequests;
 
 export const RegisterNewUser: React.FunctionComponent = () => {
-
     const history = useHistory();
     const [completedCreation, setCompletedCreation] = React.useState(false);
     const [createUser, { loading, error }] = useMutation(CREATE_USER,
         { onCompleted: () => setCompletedCreation(true) }
     );
 
+    // Get list of organisations from server
+    const { loading: getorgsloading, error: getorgserror, data: getorgsdata } = useQuery(GET_ORGANISATIONS);
+    if (getorgsloading) { return <p>Loading..</p>; }
+    if (getorgserror) { return <p>ERROR: please try again.</p>; }
+    const orgList: Models.IOrganisation[] = getorgsdata.getOrganisations;
+    const orgsMap : Record<string, string> = {};
+    for (const each of orgList) {
+        orgsMap[each.id] = each.name;
+    }
+
```

Comment:
Same as `userDetails.tsx`

Reply:
@fguitton  Done, please check the PR again :+1: